lib/js-yaml/dumper.js

```diff
@@ -569,9 +569,8 @@ function writeFlowMapping(state, level, object) {
       pairBuffer;
 
   for (index = 0, length = objectKeyList.length; index < length; index += 1) {
-    pairBuffer = state.condenseFlow ? '"' : '';
-
-    if (index !== 0) pairBuffer += ', ';
+    pairBuffer = index === 0 ? '' : ', ';
```

Comment:
Please avoid such notation, it's not enough easy to read. Use `if (index !== 0) pairBuffer += ', ';` and so on in appropriate places.